setup.py

```diff
@@ -20,8 +18,8 @@ def readme():
 )
 
 vcs = (
-    'gitpython>=2.1.8',
-    'P4'
+    'gitpython>=3.0.5',
+    'p4python>=2019.1'
```

Comment:
Dows P4 package have different names between python 2 and 3?

Reply:
yes, the correct way to install p4python now is a **pip install p4python**. It's work OK for python3.7 now.